tests/ManageCourses.Tests/ImporterTests/UcasDataMigratorTests.cs

```diff
@@ -116,6 +117,85 @@ public void ErroneousCourseLeavesOtherCoursesAlone()
             Context.Courses.Single().Name.Should().Be("The best title");
             Context.Courses.Single().Provider.Postcode.Should().Be(InstPostCode1);
         }
+
+        [Test]
+        public void DoesntImportOptedInProviders()
```

Comment:
So what happens to a course that has a accredited provider that has opted-in?

Reply:
oh

Reply:
it's okay, we aren't deleting any providers, and you can't opt-in any that aren't in our database.